multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -91,17 +93,18 @@ coreos:
       content: |
         [Unit]
         Description=decrypt kubelet tls assets using amazon kms
-        Before=kubelet.service
-        After=docker.service
-        Requires=docker.service
+        Before=kubelet.service flanneld.service
```

Comment:
Do we need this when flanneld after/requires decrypt already?  Should we add an After=decrypt to the kubelet? 

Looking around I see a mixture of use of RequiredBy vs Required.  I think they're equivalent and having units specify their dependencies makes more sense than a units specify who depends on them.  Am I wrong?  If not I'd like to standardize on Requires/After.